org.eclipse.swtchart.cbi/pom.xml

```diff
@@ -293,6 +288,30 @@
             </plugins>
          </build>
       </profile>
+      <profile>
+        <id>javadoc</id>
+          <build>
+            <plugins>
+              <plugin>
+                  <groupId>org.apache.maven.plugins</groupId>
+                  <artifactId>maven-javadoc-plugin</artifactId>
+                  <executions>
```

Comment:
Please add <version>3.11.2</version> . You have set the version in another profile and it has no effect here.  3.11.3 doesn't work for me due to https://github.com/apache/maven-javadoc-plugin/pull/1217